### PR TITLE
Add njs as a target/profile

### PR DESCRIPTION
**Pull request by 0xbigshaq**
Hi Samuel/Carl,
I took a small side-quest few weeks ago(for fun) and used _fuzzilli_. It led to writing [this](https://github.com/0xbigshaq/njs-vr) thingy, the bugs were reported in:
* https://github.com/nginx/njs/issues/713
* https://github.com/nginx/njs/issues/712

This is my attempt to give back :D thank you for creating this project.

This PR adds [njs](https://nginx.org/en/docs/njs/) as a new target/profile to fuzz so everyone can enjoy hehe

>_Note_: The logic behind the hot-patching 'technique' I implemented here is: instead of using a `.patch` files - we leverage the way they designed their build system to 'extend' the language(add `fuzzilli()` builtin, make our own shell for REPRL, etc.). I found that approach more reliable and work with more revisions.

### CODE REVIEW

File: Targets/njs/README.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+# Target: NJS
+
+To build njs for fuzzing:
+* Step 1 - prepare env
+    * Run `fuzzbuild.sh`, this will:
```
Comment (saelo) [inline Targets/njs/README.md:5]: So typically, `fuzzbuild.sh` just takes care of performing the build, not downloading the source code. You could rename this to maybe `setup.sh` or so, and then put steps 2 and 3 into a `fuzzbuild.sh`. That would be more consistent. No strong opinion though, this is also fine.

**Review by saelo — COMMENTED** [review]
Really cool, thanks for sharing! And sorry for the slow review... just a nit/suggestion the rest looks good!

File: Sources/FuzzilliCli/Profiles/NjsProfile.swift [hunk 2]
```diff
@@ -0,0 +1,59 @@
+// Copyright 2020 Google LLC
```
[the remark this answers is not in the document]
Comment (0xbigshaq) [reply Sources/FuzzilliCli/Profiles/NjsProfile.swift]: whoops, fixed :^)

**Comment by saelo** [comment]
Thanks again!

**Pull request merged**